### ISSUE TITLE
🐛 Bug Report: Placeholder Shown in the Allowed File Extensions Input

### ISSUE THREAD

**Issue by ItzNotABug** [appwrite/console#1018]
### 👟 Reproduction steps

1. Go to your Project on Console
2. Go to Storage > Settings Tab
3. Scroll to **Allowed file Extensions** and Click on the extensions below to add them
4. Notice that the placeholder text i.e. "Allowed file extensions (mp4, jpg, pdf, etc.)" moves to the right of the items added

### 👍 Expected behavior

Hide, remove the placeholder when the input is not empty.

### 👎 Actual Behavior

Placeholder is shown even when the input is not empty.

### 🎲 Appwrite version

Appwrite Cloud

### 💻 Operating system

Linux

### 🧱 Your Environment

Reproducible on `Cloud` & `1.5.x`

### 👀 Have you spent some time to check if this issue has been raised before?

- [X] I checked and didn't find similar issue

### 🏢 Have you read the Code of Conduct?

- [X] I have read the [Code of Conduct](https://github.com/appwrite/.github/blob/main/CODE_OF_CONDUCT.md)

**Comment by sourabpramanik** [comment]
That is just a helper text I don't see a bug in that, it is moreover an improvement rather than a bug imo

**Comment by ItzNotABug** [comment]
@sourabpramanik A helper text or a placeholder is usually hidden when an input contains some form of text. This one keeps moving to the right which isn't something standard imo.

**Comment by sourabpramanik** [comment]
> @sourabpramanik A helper text or a placeholder is usually hidden when an input contains some form of text. 

Agreed for text type input fields this is not the right approach but for multi-select options like this one, this makes more sense.
Maybe we can add a sub-label along with the placeholder

**Comment by FatumaA** [comment]
Can reproduce and would like to work on it if it is deemed a bug as above discussion

**Comment by Bhaaru2** [comment]
Hi! I’d like to work on this issue. If it’s still open, I can submit a fix for it.
